EnglishTranslation-master/ITEM.tsv

```diff
@@ -10288,13 +10288,13 @@ ITEM_20160310_010287	An amulet made by Lemija. It can absorb the magic occupying
 ITEM_20160310_010288	Assistant Commander's Note
 ITEM_20160310_010289	A note by the Assistant Commander of Vedas Plateau. It describes the situation of the region.
 ITEM_20160310_010290	Veda Plateau Soldier's Belongings
-ITEM_20160310_010291	Objects obtained from monsters. They seem to belong to the soldier.
+ITEM_20160310_010291	Objects obtained from monsters. They seem to be souvenirs for the soldier.
 ITEM_20160310_010292	Engagement Ring
-ITEM_20160310_010293	A ring given by Veda Plateau's Assistant Commander. Deliver it to Royal Army Officer Retia.
+ITEM_20160310_010293	A ring given by the Assistant Commander of Vedas Plateau. Deliver it to Royal Army Officer Retia.
```

Comment:
Above, it's worded the same way.  Pretty sure this is how it's supposed to be, between the two?